activerecord/lib/active_record/railties/databases.rake

```diff
@@ -229,7 +229,7 @@ db_namespace = namespace :db do
   end
 
   namespace :schema do
-    desc 'Create a db/schema.rb file that is portable against any DB supported by AR'
+    desc 'Create a db/schema.rb file that is portable against any DB supported by ActiveRecord'
```

Comment:
@arvindmehra this is valid case. You just need to put a space in there. That's what @senny meant. The same goes for that other place where there's a space missing.